src/server.c

```diff
@@ -7305,6 +7305,7 @@ struct redisTest {
     {"listpack", listpackTest},
     {"kvstore", kvstoreTest},
     {"ebuckets", ebucketsTest},
+    {"bitmap", bitopsTest},
```

Comment:
we can forward declare `int bitopsTest(int argc, char **argv, int flags);` nearby, so there is no need to add `bitops.h`.

Reply:
But no matter what, vset won't include this, right?